### PR TITLE
Use service navigation component

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
I had to try it on something! Unsure if should continue using product name, or move name of site into service header. One nice thing by doing so is that the X-GOVUK logo can now link to that website.

Screenshot of header using service navigation component.

### CODE REVIEW

**Comment by paulrobertlloyd** [comment]
Argh, Prettier went to town on this. Might need to undo some of its opinions 🙄

File: app/views/includes/_header.html [hunk 1]
```diff
@@ -1,32 +1,55 @@
...
-      <li class="x-govuk-primary-navigation__item{{ ' x-govuk-primary-navigation__item--current' if section == "organisation" }}">
-        <a class="x-govuk-primary-navigation__link" href="/organisation"{{ ' aria-current="page"' if section == "organisation" }}>By organisation</a>
-      </li>
-      <li class="x-govuk-primary-navigation__item{{ ' x-govuk-primary-navigation__item--current' if section == "contribute" }}">
-        <a class="x-govuk-primary-navigation__link" href="/contribute"{{ ' aria-current="page"' if section == "contribute" }}>Contribute</a>
-      </li>
-    </ul>
+    <div class="govuk-service-navigation__container">
+      <span class="govuk-service-navigation__service-name">
+        <a href="/" class="govuk-service-navigation__link">Services List</a>
```
Comment (frankieroberto) [inline app/views/includes/_header.html:18]: Lowercase 'l' I think?

```suggestion
        <a href="/" class="govuk-service-navigation__link">Services list</a>
```

**Review by frankieroberto — COMMENTED** [review]
@paulrobertlloyd I like this! The only thing that seems odds to me, but maybe it's ok, is that the main isn’t in the nav, except as the "Services List" item, which I didn’t initially think was clickable, and doesn't have an active state. Maybe that's ok though?

Any reason not to upgrade govuk-frontend and use the Nunjucks macro?

File: app/views/includes/_header.html (hunk 1 above)
Comment (paulrobertlloyd) [reply app/views/includes/_header.html]: I sort of thing of products being ‘Title Case’ (i.e. GOV.UK Forms, GOV.UK Design System) and services being ‘Sentence case’ (i.e. Apply for a thing).

I’ve updated the PR to keep the service name in the header, but capitalised it according to these ‘rules’ (which are also followed by other X-GOVUK sites).

**Comment by paulrobertlloyd** [comment]
> Any reason not to upgrade govuk-frontend and use the Nunjucks macro?

Doh! I think I got mislead as the header and footer are HTML (but that’s because we need to remove a lot of GOV.UK details from them).

Have updated/reverted the location of links to where the where previously, so this is more of a straight swap.

Also spotted a few bits of CSS we can now remove, too.

Screenshot of updated header and navigation.

**Review by frankieroberto — APPROVED** [review]
Looks good as a first step!

Could move the service name from the black header down to the service navigation as a follow-up?

**Comment by paulrobertlloyd** [comment]
> Could move the service name from the black header down to the service navigation as a follow-up?

Probably want to have a think how this all applies across all the sites in X-GOVUK land, where most of the projects are akin to products, not services.

**Pull request merged**